.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!-- Thanks for submitting a pull request! Please provide enough information so that others can review your pull request. -->
+
+## Summary
+
+<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
+
+
+## Test plan
+
+<!-- Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI. -->
+
+
+## Does this PR introduce a breaking change?
+
+- [ ] Yes
+- [ ] No
+
+<!-- If this PR contains a breaking change, please describe the impact and migration plan -->
+
+
+## Other information
```

Comment:
I think we should also mention here that an entry for the Changelog is desirable

Reply:
agree

Reply:
Maybe more explicit:
```md
## Changelog
- [ ] I added an entry to `CHANGELOG.md`

<!-- Please add an entry to CHANGELOG.md, summarizing the change of this PR and add them to the according section under "master": -->
<!-- * <short-summary> ([#<pr-nr>](https://github.com/thymikee/jest-preset-angular/pull/<pr-nr>)) -->
```

Reply:
I like that one, already added in